mozregression/cli.py

```diff
@@ -353,6 +361,26 @@ def check_inbounds(options, fetch_config, logger):
                                  " and --bad-rev must be set")
 
 
+def check_taskcluster(options, fetch_config, logger):
+    if not options.artifact_name:
+        raise MozRegressionError('--artifact-name is required for taskcluster'
+                                 ' regression finding')
+    try:
+        # Try an inbound-style revision range first
+        check_inbounds(options, fetch_config, logger)
+        print "Using revs: ", options.last_good_revision, options.first_bad_revision
+    except:
```

Comment:
Hm, I am not really a fan of except without exception type, since this can catch KeyboardInterrupt and such. Maybe replace the try with:
```
if options.last_good_revision and options.first_bad_revision:
    ...
```